src/ui/screens/settings/ViewSettingsMenu.qml

```diff
@@ -152,6 +167,9 @@ MZViewBase {
                 }
                 Layout.alignment: Qt.AlignHCenter | Qt.AlignBottom
                 Layout.topMargin: MZTheme.theme.vSpacingSmall - parent.spacing
+                onClicked: {
+                    Glean.interaction.signOutSelected.record({screen:telemetryScreenId})
+                }
```

Comment:
There are a few other Glean events recorded in this View, such as `settingsViewOpened` and `manageAccountClicked`. They do the exact sae thing as some of the new events. Would mind double checking with Santiago if we can just delete these old events in favour of the new ones?